docs/deployment.md

```diff
@@ -0,0 +1,13 @@
+# Deployment
+
+We build a docker image (found at docker hub as `wpengine/lostromos`) for external usage. Running
+`docker pull wpengine/lostromos` will get you the image for use.
+
+## GOOS Environment Variable
+
+In order for `lostromos` to work in the scratch container (and therefore be as small as possible), we need to build the
+binary with the normal `make build` command, but having set GOOS=linux.
+
+## Tagging
+
+We do a docker build on all pull requests, but only tag and push a functional working master with ':latest'.
```

Comment:
Shouldn't we do some kind of release tagging other than latest?  Deploying with latest tags is not really a best practice.

Reply:
I agree here. I put some other comments in place to support using our version tags.